.github/workflows/go.yml

```diff
@@ -4,11 +4,11 @@ on:
   push:
     branches:
     - main
-    - v1
+    - v1.1
```

Comment:
I think we can remove this now (and the tests will still run on this PR); I think it's also fine to proceed with merging this and removing the v1.1 tests in a subsequent PR.